lib/index.js

```diff
@@ -31,6 +45,57 @@ internals.implementation = function (server, options) {
                 return reply(Boom.unauthorized(null, 'Access Token'));
             }
     
+            var parts = parseCookies(authorization);
```

Comment:
Oh you're right @timotgl, thank you for tip, I was searching for this in hapijs documentation but didn't found it in the first place, I'll make the change.

About 2. I'll thought it would be useful, but honestly I'm happy with 'access_token' ATM and too noob to evaluate if I'm able to make this change, will see if it's easy or not